.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+charset = utf-8
+indent_style = space
+indent_size = 4
+insert_final_newline = true
```

Comment:
This rule is being broken here: 
![image](https://user-images.githubusercontent.com/1157864/185190565-3f80204b-a2b5-43e9-968e-1fbe4cb820cc.png)